docs/technical/installing-streetmix.rst

```diff
@@ -101,174 +73,176 @@ You may already have some of these prerequisites installed. Skip or update the p
 
-.. _install-linux:
+   - **Method 1: install with nvm (recommended).** :code:`nvm` is a command-line tool to manage multiple Node.js versions on a machine. This is effective for development and troubleshooting with multiple Node.js versions, and also makes it easy to install and upgrade Node.js versions over time. `Learn how to install and use nvm from the repository: https://github.com/nvm-sh/nvm <https://github.com/nvm-sh/nvm>`_.
 
-On Linux
-----------
+   - **Method 2: package manager installation.** Many Linux distributions have package managers from where you can install Node.js. The Node.js documentation provides `a list of package managers and installation instructions <https://nodejs.org/en/download/package-manager/>`_, but please note the disclaimer that package manager versions are maintained by the Linux community, not by the Node.js team. As a result, not all package managers have the latest Node.js versions at all times.
 
-The primary requirements for this project are Node.js, PostgreSQL and PostGIS. You will need those installed if you do not have them already.
+   - **Method 3: source download.** `Download the latest source tarball from the Node.js homepage <https://nodejs.org/>`_ or `choose a specific package from the Node.js download page <https://nodejs.org/download/>`_. Instructions for installing from source are sparse and is only recommended for expert Node.js users.
+
+2. Install PostgreSQL. From the `PostgreSQL download page <https://www.postgresql.org/download/>`_ select Linux, then your Linux distribution, for installation instructions.
 
-`Installing Node.js <https://www.ostechnix.com/install-node-js-linux/>`_
+3. Install PostGIS. `Linux instructions (per distribution) are available here <https://postgis.net/install/>`_.
 
-`Installing Postgres <http://postgresguide.com/setup/install.html>`_
+4. Configure PostgreSQL. If this is your first time installing PostgreSQL, you may need to set up some initial configuration, in the next section.
 
-`Installing PostGIS <https://postgis.net/install/>`_
 
-If you haven't installed Postgres on your machine before, you may need to set up some intial configuration.
+PostgreSQL configuration
+++++++++++++++++++++++++
 
 ArchLinux
-+++++++++
-`Here is an example for ArchLinux <https://wiki.archlinux.org/index.php/PostgreSQL>`_.
+~~~~~~~~~
+
+Here is `additional setup instructions for ArchLinux <https://wiki.archlinux.org/index.php/PostgreSQL>`_.
 
-Ubuntu
-++++++
-If you've just installed Postgres:
+Debian/Ubuntu
+~~~~~~~~~~~~~
 
-1. Add a user with sufficient permissions to Postgres:
+By default, installing the Debian or Ubuntu package restricts the authentication methods such that one must set up a username with a password in order for Streetmix to be able to access the database. Here are basic instructions for setting up a new user after installing PostgreSQL.
```

Comment:
I'm not sure this is entirely correct. See https://www.digitalocean.com/community/tutorials/how-to-install-and-use-postgresql-on-ubuntu-20-04 as an example (check out step 2). If this is the case, it would be best for us to be able to cite a source saying so. 

More examples here: 
https://www.postgresql.org/docs/12/auth-pg-hba-conf.html#EXAMPLE-PG-HBA.CONF

Search in the page for 'Allow any user from any host with IP address 192.168.93.x to connect' and you'll see the example connection which also the default. The default auth style is 'ident', which basically means 'your user name is the same name as your operating system name'. 

So really, default setup should be quite simple. It is possible that Ubuntu deviates from this, but I wasn't able to find any evidence of this online.

Reply:
I was leaning on the original instructions (for Ubuntu) as evidence that this is the case. Those instructions told us to set up the username and password. My system is Linux Mint, which is based on Ubuntu, and it had the exact same situation. This was the thing I was grappling with last week: try as I might, I could not make the username and password requirement go away, _unless_ I changed `pg_hba.conf` to use `trust` for the authentication method. The Ubuntu package does not install this way. 

I'll revise this slightly to hedge against the possibility that I'm wrong, or in the case that this requirement could change further down the line.